### PR TITLE
docs(ixlib): clarify disabling ixlib parameter

### PR DESCRIPTION
**Pull request by atlawrie**
Adds additional detail to the `ixlib` section of the Gatsby documentation, clarifying the differences between `includeLibraryParam` and `disableIxlibParam`, as users has expressed confusion on this point.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1117,9 +1117,9 @@ If `url` is:
 
 ### What is the `ixlib` Param on Every Request?
 
-For security and diagnostic purposes, we tag all requests with the language and version of library used to generate the URL.
+For security and diagnostic purposes, we tag all requests with the language and version of library used to generate the URL. To disable this, we provide two options. For disabling the `ixlib` parameter across all requests, we provide `disableIxlibParam` as a plugin option for use in `gatsby-config.js`.
```
Comment (luqven) [inline README.md:1120]: Maybe it's worth adding a little example code here as we do below for `indludeLibraryParam`?

**Review by luqven — CHANGES_REQUESTED** [review]
This is looking good! 

I think it could use a PR description (so if we ever need to we can look back and understand why this change was necessary) and a code example for the config file. 

Let me know what you think!

File: README.md (hunk 1 above)
Comment (atlawrie) [reply README.md:1120]: Agreed, added to the PR.

File: README.md [hunk 2]
```diff
@@ -1117,9 +1117,26 @@ If `url` is:
...
 
-To disable this, set `includeLibraryParam` in the third parameter to `false` when calling one of the two functions this library exports. For example, for `buildFluidImageData`:
+```js
+module.exports = {
+  //...
+  plugins: [
+    // your other plugins here
+    {
```
Comment (luqven) [inline README.md:1127]:
```suggestion
```js
// gatsby-config.js
module.exports = {
  //...
  plugins: [
    // your other plugins here
    {
```

File: README.md [hunk 3]
```diff
@@ -1117,9 +1117,26 @@ If `url` is:
 
 ### What is the `ixlib` Param on Every Request?
 
-For security and diagnostic purposes, we tag all requests with the language and version of library used to generate the URL.
+For security and diagnostic purposes, we tag all requests with the language and version of library used to generate the URL. To disable this, we provide two options. For disabling the `ixlib` parameter across all requests, we provide `disableIxlibParam` as a plugin option for use in `gatsby-config.js`.
```
Comment (luqven) [inline README.md:1120]:
```suggestion
For security and diagnostic purposes, we tag all requests with the language and version of the library used to generate the URL. To disable this, we provide two options. 

For disabling the `ixlib` parameter across all requests, we provide `disableIxlibParam` as a plugin option for use in `gatsby-config.js`.
```

**Review by luqven — APPROVED** [review]
LGTM aside from two nit-picks. I leave it up to you whether or not to address them before merging in.

cartoon-thumbs-up-gif-by-disney-pixar

**Pull request merged**